hat/app/org/hatdex/hat/she/service/FunctionService.scala

```diff
@@ -134,18 +134,24 @@ class FunctionService @Inject() (
       .map(_.get)
   }
 
-  def run(configuration: FunctionConfiguration, startTime: Option[DateTime])(implicit hatServer: HatServer): Future[Done] = {
+  def run(configuration: FunctionConfiguration, startTime: Option[DateTime], useAll: Boolean)(implicit hatServer: HatServer): Future[Done] = {
```

Comment:
I dont have domain knowledge so cant answer this its more of a question
Is there a better name than useAll (ignoreLast etc) ? 

    ‘There are only two hard things in Computer Science: cache invalidation and naming things.’
    - Phil Karlton

Reply:
I don't like the name too much myself. But under the time pressure could not come up with anything better. It's hard to concisely define the behaviour opposite to `useAll`, `ignoreLast` unfortunately wouldn't work. I'm open to any further suggestions to change it.

Reply:
I this example, would `useAllAvailableData` be better?

Reply:
Semantics of "AllAvailableData" is not entirely accurate in this case either. Also I think it's too long of a name for query param. Keep them coming :)